auth-server/src/utils/getMultisigAddress.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-2020 @Premiurly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { u8aSorted } from '@polkadot/util';
+import { blake2AsU8a, decodeAddress, encodeAddress } from '@polkadot/util-crypto';
+
+const derivePubkey = (addresses: string[], threshold = 1): Uint8Array => {
```

Comment:
Out of curiosity, is there any reference doc for what is being done here? This looks magical :D

Reply:
It is taken from  @w3f/msig-util for calculating multisig address from signatories

Reply:
https://wiki.polkadot.network/docs/learn-accounts